lexicons/app/bsky/administration/moderationAction.json

```diff
@@ -0,0 +1,21 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.administration.moderationAction",
+  "defs": {
+    "view": {
+      "type": "object",
+      "required": ["id", "action", "subject", "rationale", "createdBy", "createdAt"],
+      "properties": {
+        "id": {"type": "integer"},
+        "action": {"type": "string", "knownValues": ["takedown"]},
```

Comment:
Do we want to use nsids for the actions?

Reply:
Sure!  P.S. the other actions I imagined in the nearterm are a. flag (noted as questionable content/behavior, could be used as a paper trail and/or to help admins implement a policy such as three strikes) and b. acknowledge (explicit check and non-action, i.e. in response to a report).